pkg/agent/openflow/pipeline.go

```diff
@@ -2062,7 +2062,9 @@ func (c *client) generatePipeline() {
 		EgressMetricTable:     bridge.CreateTable(EgressMetricTable, l3ForwardingTable, binding.TableMissActionNext),
 		l3ForwardingTable:     bridge.CreateTable(l3ForwardingTable, l2ForwardingCalcTable, binding.TableMissActionNext),
 		l3DecTTLTable:         bridge.CreateTable(l3DecTTLTable, l2ForwardingCalcTable, binding.TableMissActionNext),
-		l2ForwardingCalcTable: bridge.CreateTable(l2ForwardingCalcTable, conntrackCommitTable, binding.TableMissActionNext),
+		// Packets from l2ForwardingCalcTable should be forwarded to IngressMetricTable by default to collect ingress stats.
+		// Setting conntrackCommitTable as nexttable will miss the packets count to the tunnel or gateway
```

Comment:
Please use `make fmt` to format it. It's required as there is a comment here so above lines don't have to keep align with L2067.

The comment at L2065 is enough to remind people why it should be IngressMetricTable, L2066 is not necessary as we won't see "conntrackCommitTable" in this line any more after it's updated.

You don't need to append "(#1976)" in the commit message title. The PR link will be added to title automatically when it's merged.

Reply:
commit message link removed and `make fmt` the code.

Leaving comment L2065 alone looks strange to me because no extra information was provided by this comment.  People can get the same information when reading the code below.

Reply:
It's to avoid people questioning why it bypasses ingress rule but not ingress metric rule and changing it back to conntrackCommitTable in the future (it's written in that way before, so people will likely feel that's right and optimize it when refactoring)

Reply:
sounds reasonable, thanks